src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -72,49 +84,64 @@ protected Uni<CollectionSettings> getCollectionProperties(
                   // TODO This if block can be deleted? grpc code
                   // ignoring the error and return false. This will be handled while trying to
                   //  execute the query
+                  // TODO: WHY ARE WE IGNORING THE ERROR AND RETURNING FAKE COLLECTION SCHEMA ? This
+                  // is a bad practice
```

Comment:
100%. Should have comments explaining why it's done (I'm fairly sure there was an actual reason, but not easy to guess from just code). But also note the reference to gRPC -- so code would only matter with old Bridge-access to Coordinator; should not be needed at all AFAIK.

Reply:
cc @Yuqi-Du ^^^ looks like there's left-over gRPC related code here, to be removed (PR against main branch)?

Reply:
Looks like just some old grpc code, yes entire if statement can be gone. Let me create a PR.

Reply:
As per comments, removed with a follow-up PR, not here.